Bottom: b10425261426b06c46c47498dd31d6de24ba3d7c
Top:    e79c1397f8c2f947fcebb3679eb61e3d19314ac3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-27 21:16:15 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
